modules/monitoring-accessing-third-party-monitoring-uis.adoc

```diff
@@ -0,0 +1,12 @@
+// Module included in the following assemblies:
+//
+// * monitoring/accessing-third-party-monitoring-uis-and-apis.adoc
+
+:_content-type: PROCEDURE
+[id="accessing-third-party-monitoring-uis"]
+= Accessing third-party monitoring UIs
+
+[role="_abstract"]
+{product-title} does not provide or support direct access to third-party web user interfaces (UIs) for components in the monitoring stack such as Prometheus, Grafana, Alertmanager, or Thanos Querier.
```

Comment:
we missed the deadline for Prometheus so it's going to be possible accessing its UI in 4.10.

Reply:
fixed